test/functional/Java17andUp/src_170/org/openj9/test/jep389/downcall/DuplicateMixedCallTests.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright IBM Corp. and others 2024
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ */
+package org.openj9.test.jep389.downcall;
+
+import org.testng.annotations.Test;
+import org.testng.Assert;
+import org.testng.AssertJUnit;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodType;
+import java.lang.invoke.VarHandle;
+
+import jdk.incubator.foreign.Addressable;
+import jdk.incubator.foreign.CLinker;
+import static jdk.incubator.foreign.CLinker.*;
```

Comment:
Fixed.

Reply:
There are several other files which use wildcard imports: please fix them as well.

Reply:
If so, we have to explicitly import all constant defined for primitives as listed in https://docs.oracle.com/en/java/javase/17/docs/api/jdk.incubator.foreign/jdk/incubator/foreign/CLinker.html (also applies to other java version in a similar way)
e.g.
```
import static jdk.incubator.foreign.CLinker.C_CHAR;
import static jdk.incubator.foreign.CLinker.C_DOUBLE;
import static jdk.incubator.foreign.CLinker.C_FLOAT;
import static jdk.incubator.foreign.CLinker.C_INT;
...
```
Is this something we expect to be there in terms test cases?

Reply:
I fixed part of files which only require importing a few constants and leave the rest of them (with all constants required) unchanged to avoid a bunch of imports.

Reply:
The point of explicitly naming each import is that it allows a reader to understand where each name comes from without having to open every type that is the target of a wildcard. So, in the context of your example, it is clear what `C_CHAR` refers to without having to consult `CLinker` or anywhere else. It just gets worse if there are multiple wildcard imports.

Reply:
Fixed the imports everywhere.